sdk/src/connections/target/addi9036_sensor.cpp

```diff
@@ -704,6 +707,10 @@ aditof::Status Addi9036Sensor::getFrame(uint16_t *buffer) {
 aditof::Status Addi9036Sensor::readAfeRegisters(const uint16_t *address,
                                                 uint16_t *data, size_t length) {
     using namespace aditof;
+
+    assert(address != NULL);
+    assert(length >= 0);
```

Comment:
length is of type size_t which an unsigned integer. This means length cannot, by definition, be less than 0.